config/routes.rb

```diff
@@ -13,6 +13,8 @@
   post '/accept_terms', action: 'accept_terms', controller: 'users', as: 'accept_terms'
 
   resources :practices, except: :index do
+    get '/edit/metrics', action: 'metrics', as: 'metrics'
+    get '/edit/practice_name', action: 'practice_name', as: 'practice_name'
```

Comment:
do we need this? since clicking on the practice name is just going to the introduction section... if we don't we can get rid of any mentions to the practice_name action

Reply:
No - we don't I removed it.